packages/manager/modules/pci/src/projects/project/storages/databases/database/advanced-configuration/advanced-configuration.html

```diff
@@ -0,0 +1,157 @@
+
+                                            <oui-select
+                                                disabled="$ctrl.pending"
+                                                name="field-{{$index}}"
+                                                ng-if="$ctrl.model[$index].key.values"
+                                                data-model="$ctrl.model[$index].value"
+                                                data-items="$ctrl.model[$index].key.values"
+                                                class="oui-select__input"
+                                            >
+                                            </oui-select>
+                                        </oui-field>
+                                    </div>
+                                </div>
+                            </fieldset>
+                            <footer
+                                class="oui-inline-adder__footer"
+                                ng-if="!$ctrl.model[$index].isFixed"
+                            >
+                                <button
+                                    type="button"
+                                    class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                    data-ng-click="$ctrl.onAddConfig($index)"
+                                    data-ng-if="!$ctrl.model[$index].added"
+                                    data-ng-disabled="!$ctrl.model[$index].key || $ctrl.addableProperties.length <= 1 || $ctrl.pending"
+                                >
+                                    <span
+                                        class="oui-icon oui-icon-add"
+                                        aria-hidden="true"
+                                    ></span>
+                                </button>
+                                <button
+                                    data-ng-disabled="$ctrl.pending"
+                                    type="button"
+                                    class="oui-inline-adder__action oui-button oui-button_primary oui-button_s"
+                                    data-ng-click="$ctrl.onRemoveConfig($index)"
+                                    data-ng-if="$ctrl.model[$index].added"
+                                >
+                                    <span
+                                        class="oui-icon oui-icon-trash"
+                                    ></span>
+                                </button>
+                            </footer>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="col-lg-6">
+            <div class="sticky-top">
+                <!-- error messages -->
+                <cui-message-container
+                    data-messages="$ctrl.messages"
+                ></cui-message-container>
+                <pre class="oui-background-p-800 oui-color-p-050">
```

Comment:
Question: you need this? you need to show the json file on UI?

Reply:
Yes, the user can see his advanced configuration as a json too:
![image](https://user-images.githubusercontent.com/24541481/157675158-83c3aa46-478e-4066-ae0b-8fe6bd6ad220.png)